README.md

```diff
@@ -0,0 +1,198 @@
+# GCP Dataflow Example Project
+
+## Introduction
+
+This is a simple stream processing example project, written in Scala and using Google Cloud Platform's services/APIs:
+
+- Dataflow for data processing
+- Pub/Sub for communication
+- Bigtable for storage
+
+**Running this requires a Google account and enabling GCP services, which will incur certain costs**
+
+We assume you have [pyinvoke](http://www.pyinvoke.org/) installed, as well as the [Google Cloud Python SDK](https://pypi.python.org/pypi/google-cloud), in order for the helper script to work.
+
+## Overview
+
+We have implemented a super simple analytics-on-write stream processing job using Dataflow. Our Dataflow job reads a Pub/Sub topic containing events in a JSON format:
+
+```json
+{
+  "timestamp": "2015-06-05T12:54:43.064528",
+  "type": "Green",
+  "id": "4ec80fb1-0963-4e35-8f54-ce760499d974"
+}
+```
+
+Our job counts the events by `type` and aggregates these counts into 1-minute buckets. The job then takes these aggregates and saves them into a table in Bigtable.
+
+## Tutorial
+
+### 1. Setting up GCP
+
+First we need to create and setup a GCP project. To do so, you can follow [this guide on Snowplow's wiki](https://github.com/snowplow/snowplow/wiki/GCP:-Getting-Started). Following those instructions, you'll need to enable the following APIs/services:
+
+- Dataflow
+- Bigtable
+- Pub/Sub
+- Storage
+
+After that, you'll need to authenticate the computer where you'll be running the project using the Google Cloud SDK. To do so, run the following commands:
```

Comment:
Quite confusing that they call their CLI tools "Cloud SDK", I'd expect this name for some programming lang library. Anyway, may it's worth to clarify to not confuse newcomers?